src/components/HomePage.vue

```diff
@@ -109,6 +138,11 @@
 
   watchEffect(() => {
     localStorage.setItem('listOfItems', JSON.stringify(listItems.value));
+    if(isContentFiltered) {
```

Comment:
1. Please add space after the `if`. 
2. In case of the `if { .. }` part contains the return you don't need to add the `else`. You can change the code like this, the result will be the same:
`localStorage.setItem('listOfItems', JSON.stringify(listItems.value));`
  `if (isContentFiltered) {`
     `return;`
 ` }`
  `displayItems.value = listItems.value;`